src/main/java/fi/helsinki/cs/tmc/core/domain/submission/SubmissionResult.java

```diff
@@ -104,14 +102,6 @@ public void setError(String error) {
         this.error = error;
     }
 
-    public Validations getValidations() {
-        return validations;
-    }
-
-    public void setValidations(Validations validations) {
```

Comment:
These indeed seem to be the classes from NetBeans and thus representing the submission result, which sometimes does contain submission results.